lib/DocumentsFieldArray/tests/DocumentsFieldArray-test.js

```diff
@@ -175,151 +172,6 @@ describe('DocumentsFieldArray', () => {
     });
   });
 
-  describe('attaching a file', () => {
```

Comment:
Can you keep a basic test that just ensures that the file uploader is present [if `onUploadFile` is provided to DFA](https://github.com/folio-org/stripes-erm-components/blob/e0119dd000d98173c4ff3d119f6cd86116e4b38c/lib/DocumentsFieldArray/DocumentsFieldArray.js#L143)?

Reply:
This line https://github.com/folio-org/stripes-erm-components/blob/e0119dd000d98173c4ff3d119f6cd86116e4b38c/lib/DocumentsFieldArray/tests/DocumentsFieldArray-test.js#L89 should take care of it? shouldn't it? unless you are referring to something else here?

Reply:
Ahh yeah, true. I didn't scroll up enough.